converter/data_converter.go

```diff
@@ -52,4 +52,18 @@ type (
 		// ToStrings converts payloads object into human readable strings.
 		ToStrings(input *commonpb.Payloads) []string
 	}
+
+	DataConverterWithContext interface {
+		DataConverter
```

Comment:
I don't think there is a need to embed DataConverter interface into DataConverterWithContext. 

I would rename it to something else as it is not a DataConverter but more like mixin.